app/models/edition.rb

```diff
@@ -594,30 +594,17 @@ def valid_as_draft?
   end
 
   def editable?
-    imported? || draft? || submitted? || rejected?
+    draft? || submitted? || rejected?
   end
 
   def can_have_some_invalid_data?
-    imported? || deleted? || superseded?
+    deleted? || superseded?
   end
 
-  attr_accessor :trying_to_convert_to_draft, :has_previously_published_error
+  attr_accessor :has_previously_published_error
 
   def errors_as_draft
```

Comment:
I think we should be able to delete the `valid_as_draft?` and `errors_as_draft` methods entirely. They were only being used by imported/speed tagging things.

...which should mean we can also get rid of this smelly patch file: [lib/patches/make_initialize_dup_public.rb](https://github.com/alphagov/whitehall/blob/a7d4f4c4e222ac482a1bce7f0a32f6e68a3c6212/lib/patches/make_initialize_dup_public.rb)

Reply:
Done! Thanks @ollietreend 👍